node/core/approval-voting/Cargo.toml

```diff
@@ -6,13 +6,13 @@ edition = "2018"
 
 [dependencies]
 futures = "0.3.8"
-parity-scale-codec = { version = "1.3.5", default-features = false, features = ["bit-vec", "derive"] }
+parity-scale-codec = { version = "2.0.0", default-features = false, features = ["bit-vec", "derive"] }
 
 polkadot-subsystem = { package = "polkadot-node-subsystem", path = "../../subsystem" }
 polkadot-overseer = { path = "../../overseer" }
 polkadot-primitives = { path = "../../../primitives" }
 polkadot-node-primitives = { path = "../../primitives" }
-bitvec = "0.17.4"
+bitvec = "0.20.1"
```

Comment:
I wonder whether we only need
```toml
bitvec = { version = "0.20.1", default-features = false, features = ["alloc"] }
```
as in other places. otherwise cargo will end up using the default features everywhere.

Reply:
I guess those crate are only compiled with std